### ISSUE TITLE
SASS Variables can not be overriden using Custom Theme

### ISSUE THREAD

**Issue by taauntik** [bryntum/support#9441]
[Forum post](https://forum.bryntum.com/viewtopic.php?f=54&t=29507&p=149143#p149143)

Hey,
we are trying to implement a custome theme for the calendar following the documentation found here: [Calendar/Customization/Styling#creating a custom theme via npm.](https://bryntum.com/products/calendar/docs/guide/Calendar/customization/styling#creating-a-custom-theme)
The documentation says to override the variable and import one of the included themes. However almost all variables in the custom themes do not specify !default and can therfore not be overriden by us.

Example:

```
$fa-font-path: "@bryntum/calendar/fonts";
$roboto-font-path: "@bryntum/calendar/fonts";

$button-default-color: red; <!-- has no effect -->

@import "node_modules/@bryntum/calendar/source/resources/sass/themes/material";

```
The documentation says that all variable can be found inside 
"node_modules/@bryntum/calendar/source/resources/core-sass/themes/vars/material-vars.scss"

```
$button-default-color                    : $primary;

```

Is this intended behavior? If so, how can we create a full custom theme?

**Comment by ghulamghousdev** [comment]
Also check for the issue mentioned here: https://forum.bryntum.com/viewtopic.php?p=150033#p150033

**Comment by geop-bbm** [comment]
My temporary workaround was to modify the file "resources\core-sass\themes\vars\classic-vars.scss".
The option ‘!default’ to make $button-default-color overwritable is missing here. 

`$button-default-color                      : scale-color($gray, $lightness : 20%) !default;`

After that my custom SCSS build worked for the Buttons.

**Comment by dan-thony** [comment]
Yeah, a lot more of the variables in the themes ( like`$primary : $blue;` in material-vars.scss), if they had `!default`, would allow custom variable over-rides.

**Comment by isglass** [comment]
Closing this since themes no longer use SASS variables
